ipatests/test_integration/test_subids.py

```diff
@@ -268,3 +269,30 @@ def test_sudid_match_shows_uid(self):
                                           f"--subuid={subuid}"])
         owner = self._parse_result(result)["owner"]
         assert owner == uid
+
+    def test_nsswitch_doesnot_contain_subid_entry(self):
+        """
+        This testcase checks that when ipa-client-install
+        is installed without subid option, the nsswitch.conf
+        doesnot contain subid entry
+        """
+        cmd = self.clients[0].run_command(
+            ["grep", "^subid", "/etc/nsswitch.conf"]
```

Comment:
You need to add `raiseonerr=False` otherwise the test will fail with a `CalledProcessError` exception.